raystack/frontier/v1beta1/frontier.proto

```diff
@@ -698,6 +698,15 @@ service FrontierService {
     };
   }
 
+  rpc GetOrganizationKyc(GetOrganizationKycRequest) returns (GetOrganizationKycResponse) {
+    option (google.api.http) = {get: "/v1beta1/organizations/{id}/kyc"};
```

Comment:
the `{id}` should be available in the message body. otherwise, grpc-gateway can't link it.